### DISCUSSION TITLE
Since this project is on hiatus, suggestion below:

### DISCUSSION THREAD

**Discussion by FrostKnight** [tenacityteam/tenacity-legacy#729] [General]
For anyone who isn't aware, I know a project, that is actually still being worked on that is neither audacity nor sneed...

https://github.com/saucedacity/saucedacity

Not the best project name, but this person is still working on it since july 15 last year.

thus, I suggest anyone who still wants to rebuke and maybe crush the current audacity, join this project...

Again... this one:

https://github.com/saucedacity/saucedacity

That is all...

It is better than surrendering to the audacity shmucks isn't it?

Anywho, peace all!

**Comment by RockyMM** [comment]
If only all wars could be fought on GitHub, the world would be much more nicer place.

**Reply by FrostKnight** [reply]
On that note, perhaps, but you do who owns github right? 

For this reason, maybe... depends on the situation...

If you had said this about something like codeberg or something similar, I might have agreed completely...

Btw, on that note, it would help if people took my message above seriously, even if its still this project is back on track, or preferably, both projects work together more or less, when it seems benefitical to both, or better.
